### ISSUE TITLE
[BUG] New currencies. Price tooltip completely broken.

### ISSUE THREAD

**Issue by sharkiller** [jshackles/Enhanced_Steam#1480]
@jshackles please add support for new currencies. Tooltip comparison is completely broken.

Like Argentine Peso (ARS)
![image](https://user-images.githubusercontent.com/712193/32757268-664702e0-c8bd-11e7-8aa0-eae728caf685.png)

**Comment by schiza** [comment]
It also breaks for PLN (Poland currency) which was just introduced, so it's not working at all.

**Comment by yarrmateys** [comment]
this part doesn't work at all in poland. the globe doesn't appear in firefox nor in chrome, and the alternate lower prices don't get recalculated to native currency either, so you have a price in zł, but the lower prices are still listed in euro.

**Comment by frangua** [comment]
yes, its broken, in Firefox it doesn't show anything. Just blank.

**Comment by Cula97** [comment]
I changed the settings of currency from auto-detect to US and the icon reappeared but the prices aren't being recalculated

**Comment by VanityFox** [comment]
So, Enhanced Steam is discontinued. There is now [Augmented Steam](https://github.com/tfedor/AugmentedSteam/). Perhaps your requested feature/bugfix is already implemented there! If not, you could move your issue there.
